### DISCUSSION TITLE
JOY-IT JT-RAD01: Same hardware as FNIRSI GC-01

### DISCUSSION THREAD

**Discussion by GeigerMuller** [Gissio/radpro#156] [General]
Hi,

I thought to expand the community it would be good to add another device to the list of supported devices. 

The name **JOY-IT JT-RAD01** has exactly the same hw as the original **GC-01**. 

I have two pieces
installed cpu: CH32F103R8 
installed GM tube: 1 device: J321 / 2 device: M4011
screen cable marking: 1 device: CL24CG1045-40B / 2 device:  ENH-TV0240B026
**both devices have resistor R38 in the high-voltage circuit**

I installed (moved) RadPro from nb with win10 via usb 2.0. On the JT-RAD01 device, I had to hold the on/off button the whole time, otherwise the devices would disconnect. 
That's all. On both devices RadPro works.

![f3](https://github.com/user-attachments/assets/7e07636c-69b2-4312-b36b-4bef0402d9bb)

![f1](https://github.com/user-attachments/assets/422128c2-ddda-4676-ac6c-3c7588b72951)

![f2](https://github.com/user-attachments/assets/f7829ebc-2a46-4e83-a7d5-49547434ce49)

![fj321](https://github.com/user-attachments/assets/9db0161e-a116-47d2-b1c0-b85b67ebff8e)

![f4011](https://github.com/user-attachments/assets/1370963b-dfef-457d-84bf-dceddf84497a)

if you need more photos or some tests, don't hesitate to contact me.

**Comment by Gissio** [comment]
Thanks for the report!
